infra/chromadb_compaction/utils/logging.py

```
@@ -0,0 +1,180 @@
+"""Structured logging configuration for ChromaDB compaction operations."""
+
+import json
+import logging
+import os
+import time
+import uuid
+from contextlib import contextmanager
+from typing import Optional, Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)